optional-integration-analysis/merge-sce.R

```diff
@@ -0,0 +1,109 @@
+## This script takes in individual SCE objects and merges them into one SCE object
+## for a data integration analysis.
+
+## Set up ----------------------------------------------------------------------
+
+# Load libraries
+library(optparse)
+library(SingleCellExperiment)
+
+# Declare command line options
+option_list <- list(
+  make_option(
+    opt_str = c("--input_metadata_tsv"),
+    type = "character",
+    help = "Path to the TSV file containing the list of library IDs corresponding to the libraries being integrated, must have `library_id` and `processed_sce_filepath` columns."
+  ),
+  make_option(
+    opt_str = c("-o", "--output_sce_file"),
+    type = "character",
+    help = "Path to output RDS file, must end in .rds"
```

Comment:
```suggestion
    help = "Path to output RDS file containing merged object, must end in .rds"
```
Just to clarify what exactly is being output here.